CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.17.1 (2021-05-05)
+
+- Fixed the input box component to utilize the correct placeholder font size. [#263](https://github.com/blackbaud/skyux-forms/pull/263)
```

Comment:
```suggestion
- Fixed the input box component to use the correct font size for placeholder text. [#263](https://github.com/blackbaud/skyux-forms/pull/263)
```
These are minor tweaks that I think make this read a little better. But I'd only bother with these changes if we are going to have to rerun checks to fix the date. If not, I'd just merge this as is. I'll go ahead and approve in case you merge with the existing date, but I can re-approve if you make either suggested update.